app/mikane/src/app/features/mobile/event-item/event-item.component.html

```diff
@@ -1,7 +1,21 @@
 <mat-list-item class="wrapper" [ngClass]="{ 'ready-to-settle': event.status.id === EventStatusType.READY_TO_SETTLE }">
 	<div class="event-title">
 		<span class="title-text">{{ event.name }}</span>
-		@if (event.userInfo.inEvent) {
+		@if (event.private) {
+			<svg
+				xmlns="http://www.w3.org/2000/svg"
+				viewBox="0 -960 960 960"
+				height="24px"
+				width="24px"
+				fill="#ffffff"
+				matTooltip="This event is private"
+				[matTooltipShowDelay]="200"
+				class="joined-icon"
+			>
+				<path d="M120-160v-112q0-34 17.5-62.5T184-378q62-31 126-46.5T440-440q20 0 40 1.5t40 4.5q-4 58 21 109.5t73 84.5v80H120ZM760-40l-60-60v-186q-44-13-72-49.5T600-420q0-58 41-99t99-41q58 0 99 41t41 99q0 45-25.5 80T790-290l50 50-60 60 60 60-80 80ZM440-480q-66 0-113-47t-47-113q0-66 47-113t113-47q66 0 113 47t47 113q0 66-47 113t-113 47Zm300 80q17 0 28.5-11.5T780-440q0-17-11.5-28.5T740-480q-17 0-28.5 11.5T700-440q0 17 11.5 28.5T740-400Z"/>
```

Comment:
Surprised there isn't a material icon for this

Reply:
There is, but it doesn't work for whatever reason 🤔